cdap-docs/admin-manual/build.sh

```diff
@@ -80,9 +82,16 @@ function download_includes() {
   types="configuration hdfs-permissions"
   for type in ${types}; do
     rewrite_references_in_place_sed "${target_includes_dir}/mapr-${type}.rst" " su hdfs" " su mapr"
-    rewrite_references_in_place_sed "${target_includes_dir}/mapr-${type}.rst" "FQDN1:2181,FQDN2:2181" "FQDN1:5181,FQDN2:5181"
   done
   
+  local source_pattern="(FQDN1:2181,FQDN2:2181)"
+  local target_pattern=$(cat <<EOF
+(FQDN1:5181,FQDN2:5181);
+             note that the MapR default of 5181 is different than the ZooKeeper default of 2181
```

Comment:
i'm not super familiar with the docs build, but isn't putting doc content into build.sh kindof an anti-pattern?  I think this "note" doesn't actually have to be embedded into the <property> literal... we can probably just put it in the normal text above